tests/test_sources.py

```diff
@@ -105,7 +105,7 @@ def test_create(self):
         self.assertEqual(source.type, "arcgis")
         self.assertEqual(source.license, None)
         self.assertEqual(source.publish_to_catalog_services, False)
-        self.assertEqual(source.permissions, "https://test.koordinates.com/services/api/v1/sources/58/permissions/")
+        # self.assertEqual(source.permissions, "https://test.koordinates.com/services/api/v1/sources/58/permissions/")
```

Comment:
Oh yes, I forgot about this. Before a GET layer call to the api will include a `response` field in the returned body, something like:
```
 "permissions": "https://vm.dev.kx.gd/services/api/v1/layers/1/permissions/",
```

But now the instances (such as `Layer`)  are using the `permissions` property to access the `PermissionManager`, which causes to lose that url that was provided by the API. I don't think it's a problem since that URL doesn't need to be used by the client.